resources/lang/en/frontend.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+return [
+	'global' =>[
+		'dashboard' => 'Dashboard',
```

Comment:
Oh... I said you should have left the original strings in but just move it to PHP. We can revert the commit so you don't have to manually undo anything and I'll move the strings into the PHP files. I feel bad that you've done all this work! Lol